apps/condo/domains/settings/schema/MobileFeatureConfig.test.js

```diff
@@ -0,0 +1,217 @@
+                    const client = await makeClient()
+                    await expectToThrowAuthenticationErrorToObjects(async () => {
+                        await MobileFeatureConfig.getAll(client, {}, { sortBy: ['updatedAt_DESC'] })
+                    })
+                })
+            })
+
+
+            describe('employee', () => {
+                test('can create', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageMobileFeatureConfigs: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [obj, attrs] = await createTestMobileFeatureConfig(client, organization)
+
+                    expect(obj.id).toMatch(UUID_RE)
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.createdBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can update', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageMobileFeatureConfigs: true,
+                    })
+                    await createTestOrganizationEmployee(admin, organization, client.user, role)
+
+                    const [objCreated] = await createTestMobileFeatureConfig(admin, organization)
+
+                    const [obj, attrs] = await updateTestMobileFeatureConfig(client, objCreated.id)
+
+                    expect(obj.id).toMatch(UUID_RE)
+                    expect(obj.dv).toEqual(1)
+                    expect(obj.sender).toEqual(attrs.sender)
+                    expect(obj.v).toEqual(2)
+                    expect(obj.updatedBy).toEqual(expect.objectContaining({ id: client.user.id }))
+                })
+
+                test('can\'t delete', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+
+                    await expectToThrowAccessDeniedErrorToObj(async () => {
+                        await MobileFeatureConfig.delete(client, 'id')
+                    })
+                })
+
+                test('can read', async () => {
+                    const client = await makeClientWithNewRegisteredAndLoggedInUser()
+                    const [organization] = await createTestOrganization(admin)
+
+                    const [role] = await createTestOrganizationEmployeeRole(admin, organization, {
+                        canManageMobileFeatureConfigs: true,
```

Comment:
`canManage` is used for controlling read access?

Reply:
For now, every employee can read models (according to access rules) despite the `can` rule.

Reply:
Yes, you're right, most likely mechanically wrote.

Reply:
> If we decide in the future to restrict access for support users, the tests will not show any errors. For now, we have a separate type of user and have no test for them.

Hmm, the support now has the same read/modify rights as the admin. There is no other special logic for the support. All that can change is that rights can be taken away. But then nothing breaks. I don't think anyone does it by accident.

Reply:
added tests for support